apps/app.near/widget/Components/Profile/Tab/Videos.jsx

```diff
@@ -0,0 +1 @@
+return <div>Test</div>;
```

Comment:
Would you say that Image, Post, Videos, NFTs... are all just Feeds with unique Layout, Item, and items?

Reply:
We can say that